app/controllers/mock.php

```diff
@@ -646,6 +649,69 @@
         $response->noContent();
     });
 
+App::get('/v1/mock/github/callback')
+    ->desc('Create installation document using GitHub installation id')
+    ->groups(['mock', 'api', 'vcs'])
+    ->label('scope', 'public')
+    ->param('installation_id', '', new Text(256, 0), 'GitHub installation ID', true)
+    ->param('projectId', '', new Text(2048), 'Project ID of the project where app is to be installed', true)
+    ->inject('gitHub')
+    ->inject('project')
+    ->inject('response')
+    ->inject('dbForConsole')
+    ->action(function (string $providerInstallationId, string $projectId, GitHub $github, Document $project, Response $response, Database $dbForConsole) {
```

Comment:
At beginning of action, lets add something like this: https://github.com/appwrite/appwrite/blob/main/app/controllers/mock.php#L632-L636

That ensures the mock endpoint cannot be used on production.

We might want to ask Chrsty about this, since we maaaaybe want to disable all mock endpoints on production. So we might want to do this in `::init()` of mock group. Maybe. But maybe we actually use some mock endpoints for something like SDK tests, so please confirm with Chrsity.